adafruit_debouncer.py

```diff
@@ -148,10 +159,10 @@ class Button(Debouncer):
 
     def __init__(
         self,
-        pin,
-        short_duration_ms=200,
-        long_duration_ms=500,
-        value_when_pressed=False,
+        pin: DigitalInOut,
```

Comment:
I do think it should be typed the same as `io_or_predicate`

It may even be worth renaming `pin` to `io_or_predicate` so it matches. I wonder how much code might exist that would need to be changed, hopefully not too much since it's not a kwarg.